src/ShellSdk.ts

```diff
@@ -189,6 +189,9 @@ export class ShellSdk {
           this.debugger.traceEvent('outgoing', payload.type, payload.value, { from }, true);
           this.target.postMessage({ type: payload.type, value: payload.value, from }, this.origin);
           return;
+        } else if (source != this.target) {
```

Comment:
I see this was done in previous PR already, can we just close this one ?

Reply:
.... i was hoping to release 1.1.2 as the other PR is 1.2.0 ... we can't release outside of master ? That is something we might need to think about 😅